src/components/Auth/assets/FaceTecStrings.js

```diff
@@ -0,0 +1,124 @@
+const feedback = (el) => `auth_capture.feedback.${el}`
+const presessionFeedback = (el) => `auth_capture_start.feedback.${el}`
+export const FaceTecStrings = (t) => ({
```

Comment:
Firstly, using a single letter `t` is quite an anti-pattern in JS, because it decreases the readability.
Secondly, you're assuming the passed translate function is safe and ready for your calls, but in real life, sometimes it isn't.

```suggestion
export const FaceTecStrings = (translate) => {
  // `translate` function checks
  if (!translate || typeof translate !== 'string') {
    // ...
  }
  
  return {
    FaceTec_accessibility_cancel_button: translate('auth_accessibility.back_button'),
    // ...
  }
}
```

Reply:
Also, please convert this into TypeScript :pray: